Update README-DEV.md

thevaadinman opened the pull request:
Minor fix of a minor typo to improve initial impressions. :)

thevaadinman committed:
Update README-DEV.md

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vaadin/framework?pullRequest=12518) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vaadin/framework?pullRequest=12518) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vaadin/framework?pullRequest=12518) it.</sub>

thevaadinman committed:
Update README-DEV.md

Add missing 'the' to make wording more prosaic

thevaadinman committed:
Fix vaadin-uitest POM file errors

Maven complains about errors in the POM for uitest - "could not initialize class org.apache.maven.plugin.war.util.WebappStructureSerializer" and about the 'skip' option being incorrect for maven-javadoc-plugin (because it was missing a groupid declaration). Also silenced error for explicitly set javax.servlet-api version.

thevaadinman commented:
I accidentally hit the "push" button in Github Desktop and pushed something I didn't want. :<
